mollie/invoices.go

```diff
@@ -0,0 +1,109 @@
+package mollie
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/google/go-querystring/query"
+	"net/http"
+)
+
+type InvoiceStatus string
+
+const (
+	InvoiceStatusOpen    InvoiceStatus = "open"
+	InvoiceStatusPaid    InvoiceStatus = "paid"
+	InvoiceStatusOverdue InvoiceStatus = "overdue"
+)
```

Comment:
Also, we need a comment for `InvoiceStatus` and a block descriptive comment for the const block.